content/documentation/pages/1-installation/3-kubernetes/2-kubectl.md

```diff
@@ -0,0 +1,328 @@
+---
+path: 'installation/kubernetes/kubectl'
+title: 'kubectl'
+description: 'Installation using kubectl'
+---
+
+# Deploying with `kubectl`
+
+To install with `kubectl`, you need to get the Kubernetes configuration
+files.
+
+They have the required metadata set for service discovery needed by the different applications and services deployed.
+To check out the code, enter the following commands:
+
+    $ git clone https://github.com/{github-repo}
```

Comment:
We can uses wget for the proper files instead of the git clone option.   David's example gives us an example of this.

Reply:
there are many files, would need to have a .zip